Tasks/ExtractFiles/task.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "5e1e3830-fbfb-11e5-aab1-090c92bc4988",
+  "name": "ExtractFiles",
+  "friendlyName": "Extract Files",
+  "description": "Extract files from a variety of archive types, e.g. .zip, .tar, .7z",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "Replace with markdown to show in help",
```

Comment:
In the help Markdown, we should list every supported archive/compression format.